server/datastore/mysql/users.go

```diff
@@ -107,6 +108,20 @@ func (ds *Datastore) findUser(ctx context.Context, searchCol string, searchVal i
 	return user, nil
 }
 
+// HasUsers returns whether Fleet has any users registered
+func (ds *Datastore) HasUsers(ctx context.Context) (bool, error) {
+	var id uint
+	if err := sqlx.GetContext(ctx, ds.reader(ctx), &id, `SELECT id FROM users LIMIT 1`); err != nil {
```

Comment:
I always thought `SELECT EXISTS (SELECT 1 FROM users)` was the "fastest" way to check if a table was empty or not. But we are splitting hairs here, both are way faster than the original query 👍 🚀